spec/features/state_file/complete_intake_spec.rb

```diff
@@ -255,66 +259,63 @@
       fill_in 'state_file1099_g_state_income_tax_withheld_amount', with: "789"
       click_on I18n.t("general.continue")
 
-      expect(page).to have_text(I18n.t('state_file.questions.unemployment.index.1099_label', name: StateFileNcIntake.last.primary.full_name))
+      page_change_check(I18n.t('state_file.questions.unemployment.index.1099_label', name: StateFileNcIntake.last.primary.full_name))
       click_on I18n.t("general.continue")
 
-      expect(page).to have_text(I18n.t("state_file.questions.nc_retirement_income_subtraction.edit.title"))
+      page_change_check(I18n.t("state_file.questions.nc_retirement_income_subtraction.edit.title"))
       choose strip_html_tags(I18n.t("state_file.questions.nc_retirement_income_subtraction.edit.income_source_bailey_settlement_html"))
       check I18n.t("state_file.questions.nc_retirement_income_subtraction.edit.bailey_settlement_at_least_five_years")
       click_on I18n.t("general.continue")
 
-      expect(page).to have_text(I18n.t("state_file.questions.nc_retirement_income_subtraction.edit.title"))
+      page_change_check(I18n.t("state_file.questions.nc_retirement_income_subtraction.edit.title"))
       choose I18n.t("state_file.questions.nc_retirement_income_subtraction.edit.other")
       click_on I18n.t("general.continue")
 
-      expect(strip_html_tags(page.body)).to have_text strip_html_tags(I18n.t("state_file.questions.nc_subtractions.edit.title_html.other"))
+      sleep 0.2
+      expect(page).to have_current_path("/en/questions/nc-subtractions")
```

Comment:
the test seemed to get snagged here a lot, seemed to have to do with the `strip_html_tags`, avoided this by putting the expectation on the path name instead and adding a sleep

Reply:
why didn't you do:
```
      page_change_check(I18n.t("state_file.questions.nc_subtractions.edit.title_html.other"))
```

Reply:
I did try that but unfortunately it would still flake out right here a lot, I think because it didn't like `strip_html_tags` 😭